fuseki/sample-ubd.trig

```diff
@@ -0,0 +1,280 @@
+  .
+
+  <cube-project/ubd/csv-source/stations/column/station-id>
+    a csvw:Column ;
+    schema:name "STATION_ID" ;
+    dtype:order 0 ;
+    cc:csvColumnSample "1" ;
+    cc:csvColumnSample "2" ;
+    cc:csvColumnSample "3" ;
+  .
+
+  <cube-project/ubd/csv-source/stations/column/station-name-fr>
+    a csvw:Column ;
+    schema:name "STATION_NAME_FR" ;
+    dtype:order 1 ;
+    cc:csvColumnSample "ABC" ;
+    cc:csvColumnSample "DEF" ;
+    cc:csvColumnSample "GHI" ;
+  .
+
+  <cube-project/ubd/csv-source/stations/column/station-name-de>
+    a csvw:Column ;
+    schema:name "STATION_NAME_DE" ;
+    dtype:order 2 ;
+    cc:csvColumnSample "ABK" ;
+    cc:csvColumnSample "DEF" ;
+    cc:csvColumnSample "GHI" ;
+  .
+}
+
+<cube-project/ubd/csv-mapping/tables> void:inDataset <ubd-example> .
+graph <cube-project/ubd/csv-mapping/tables> {
+  <cube-project/ubd/csv-mapping/tables>
+    a cc:TableCollection, hydra:Collection ;
+    hydra:title "Tables" ;
+    hydra:manages [ hydra:property rdf:type ;
+                    hydra:object   cc:Table
+                  ] ;
+    hydra:manages [ hydra:property cc:csvMapping ;
+                    hydra:object   <cube-project/ubd/csv-mapping>
+                  ] ;
+    cc:csvMapping <cube-project/ubd/csv-mapping> ;
+  .
+}
+
+<project/ubd/csv-mapping/table-observation> void:inDataset <ubd-example> .
+graph <project/ubd/csv-mapping/table-observation> {
+  <project/ubd/csv-mapping/table-observation>
+    a cc:Table, cc:ObservationTable, hydra:Resource ;
+    cc:csvw <project/ubd/csv-mapping/table-observation/csvw> ;
+    cc:csvMapping <cube-project/ubd/csv-mapping> ;
+    cc:csvSource <cube-project/ubd/csv-source/ubd> ;
+    schema:name "Observations" ;
+    schema:color "#AAAAAA" ;
+    cc:identifierTemplate "ammonia/observation/{STATION}-{YEAR}-annualmean" ;
+    cc:columnMapping <project/ubd/csv-mapping/table-observation/column-mapping-1> ;
+    cc:columnMapping <project/ubd/csv-mapping/table-observation/column-mapping-2> ;
+    cc:columnMapping <project/ubd/csv-mapping/table-observation/column-mapping-3> ;
+    cc:columnMapping <project/ubd/csv-mapping/table-observation/column-mapping-4> ;
+  .
+}
+
+<project/ubd/csv-mapping/table-observation/column-mapping-1> void:inDataset <ubd-example> .
+graph <project/ubd/csv-mapping/table-observation/column-mapping-1> {
+  <project/ubd/csv-mapping/table-observation/column-mapping-1> a cc:ColumnMapping ;
+    cc:sourceColumn <cube-project/ubd/csv-source/ubd/column/year> ;
+    cc:targetProperty <dimension/year> ;
+    cc:datatype xsd:gYear ;
+    # -- Other possible ColumnMapping properties
+    # cc:language
+    # cc:default
+    # cc:datatype/params TBD
+  .
+}
+
+<table-observation/column-mapping-2> void:inDataset <ubd-example> .
+graph <table-observation/column-mapping-2> {
+  <table-observation/column-mapping-2> a cc:ColumnMapping ;
+    cc:sourceColumn <cube-project/ubd/csv-source/ubd/column/station> ;
+    cc:targetProperty <station> ;
```

Comment:
@martinmaillard I'm a little unsure about these relative property URIs as they are based in the app URI